virtcontainers/sandbox.go

```diff
@@ -543,6 +543,7 @@ func newSandbox(ctx context.Context, sandboxConfig SandboxConfig, factory Factor
 		shmSize:         sandboxConfig.ShmSize,
 		sharePidNs:      sandboxConfig.SharePidNs,
 		stateful:        sandboxConfig.Stateful,
+		networkNS:       NetworkNamespace{NetNsPath: sandboxConfig.NetworkConfig.NetNSPath},
```

Comment:
@mcastelino Looking at your description of the issue #1664 there are actually 2 ways to fix this. The network config is stored inside `sandbox.config.NetworkConfig`, so you could find the same information from the config.
But this patch looks good to me too :)

Reply:
The two are the same, -- `sandbox.config` is `sandboxConfig` as assigned above.

Reply:
@sboeuf yes the information is available within the store, and that is what I ended up doing. However it is wrong for us to pass in an empty value when there is a value present. So either we need to remove that parameter and always retrieve if from config or set it up properly.